### ISSUE TITLE
no support hardware acceleration video4linux2

### ISSUE THREAD

**Issue by stamatovg** [nicknsy/jellyscrub#103]
I'm using arm64 sbc with jellyfin hardware acceleration
Video4Linux2 (V4L2) and the cpu when play movie is under 3-4%
but if i installed jellescrub and HW Acceleration on 'Full - decode' or 'Non - decode' i have ffmpeg process with 400%
`    PID USER      PR  NI    VIRT    RES    SHR S  %CPU  %MEM     TIME+ COMMAND 
199724 911       30  10  980388  76756  15776 S 569.8   0.5  13:41.28 ffmpeg   `
Could you add hardware acceleration Video4Linux2 for jellyscrub?

**Comment by nicknsy** [comment]
Hardware acceleration in jellyscrub is essentially 1:1 with whatever jellyfin supports, so I don't do anything to add support for specific platforms. However, if the cpu usage is too high you could try setting the threads option in jellyscrub to something lower like 1 or 2.

**Comment by deadcxap** [comment]
I agree with the question. I have an intel i3-12100. Recoding in jellyfin works fine, I see that there is a load on the video core. When jellyscrub starts working, it only shows the load on the processor cores, and not on the video chip.
